src/main/java/com/pjohnson_wtc/student_db_admin/Admin.java

```diff
@@ -1,10 +1,136 @@
 package com.pjohnson_wtc.student_db_admin;
 
-public class Admin {
-
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.*;
 
+public class Admin {
+	
+	Logger logger = Logger.getLogger("com.pjohnson_wtc.student_db_admin.admin");
+	
+	private List<Student> allStudents = new ArrayList<Student>();
+	
+	//Method for creating new Student from user input
+	public int[] createStudent(InputStream inputStream) throws NumberFormatException {
+		
+		//Create new Scanner and take user input
+		Scanner scanner = new Scanner(inputStream);
+		
+		System.out.println("How many students are you adding?");
```

Comment:
No System Out statement

Reply:
Hi Puneet,

If I wasn't using a Scanner to communicate with the user via the console, I wouldn't use System.out; however, please do let me know if there's a better alternative when using Scanners.

Reply:
Logger works - woohoo! Will change them over